photos/user.go

```diff
@@ -138,8 +141,21 @@ func startStats(stopper *stop.Stopper) {
 // startUser simulates a stream of user events until the stopper
 // indicates it's time to exit.
 func startUser(ctx Context, stopper *stop.Stopper) {
+	h := fnv.New32()
+	var buf [8]byte
+
+	randomUser := func() int {
+		// User an exponential distribution to skew the user ID generation, but
+		// hash the randomly generated value so that the "hot" users are spread
+		// throughout the user ID key space (and thus not all on 1 range).
+		binary.BigEndian.PutUint64(buf[:8], math.Float64bits(rand.ExpFloat64()/rate))
+		h.Reset()
+		h.Write(buf[:8])
+		return 1 + int(h.Sum32())
```

Comment:
I bet it was trying to make sure it never generated a user ID of zero (although that doesn't really matter).